src/pywink/test/devices/api_responses/robot.json

```diff
@@ -0,0 +1,79 @@
+{
+  "robot_id": "4830303",
+  "name": "Test",
```

Comment:
Please change this name to "Test Robot" to help make the subsequent tests easier to read.